modules/optimization/doc/content/modules/optimization/examples/shapeOpt_Annulus.md

```diff
@@ -0,0 +1,122 @@
+
+In the forward app, a heat conduction problem is solved with convective flux
+type boundary condition that changes with the inner radius. The outer radius is
+insulated and there is a constant source term throughout. The objective is
+find the two radii that will have the lowest max temperature while satisfying a
+volume constraint.
+
+!listing test/tests/executioners/constrained/shape_optimization/forward.i
+         id=forward_app
+         caption= Constrained Optimization Forward SubApp
+
+In the forward app, the two optimization parameters control the inner and outer
+values are used to displace the boundaries of the
+[ParsedOptimizationFunction](/Functions/ParsedOptimizationFunction) to calculate
+the cartesian boundary displacement. These
+values are used displace the boundaries of the
+annulus to the correct position using Dirichlet boundary conditions. A diffusion
+problem on the displacements then allow for the interior elements to be
+smoothed. For the temperature field, the physics is solved on the displaced mesh
+to calculate the current objective.
+
+For the equality constraints, a gradient of the constraint with respect to the
+parameters is needed for TAO. The gradient is analytically computed for the
+current radii.
+
+!listing test/tests/executioners/constrained/shape_optimization/forward.i
+         block= Functions BCs Kernels
+         id=forward_app_shape
+         caption= Forward App Shape Optimization
+
+Using postprocesors, the objective and constraint functions can be calculated.
+The objective in this case is to minimize the maximum temperature. Also the
+current volume is calculated and how much that volume violates the constraint.
+These values are now able to be transferred to the main application for the
+optimization process.
+
+
+!listing test/tests/executioners/constrained/shape_optimization/forward.i
+         block=Postprocessors
+         id=forward_app_obj
+         caption= Forward App Objective Calculations
+
+## Adjoint Problem Sub-App
+
+Since we are using finite-differencing to determine the gradients of our
+objective function, a Adjoint Sub-App is not needed.
+
+## Optimization Results
+
+As shown in [setup], the initial annulus design has a temperature profile that
+has max temperature around 470.
+
+!media large_media/optimization/constrainedOpt_shapeSetup.png
+        style=width:60%;margin:auto;padding-top:2.5%;background-color: white;color: black;
+        id=setup
+        caption= Initial annulus temperature profile
+
+After optimizing the design to decrease the max
+temperature, the new design has a max temperature around 160 and the volume
+constraint was satisfied to the tolerance. [opt_sol] shows the optimized designs
+and the new temperature profile.
+
+!media large_media/optimization/constrainedOpt_shapeSoln.png
+        style=width:60%;margin:auto;padding-top:2.5%;background-color: white;color: black;
+        id=opt_sol
+        caption= Optimized Annulus temperature profile
```

Comment:
I would put some rulers on the figure to show what the resulting outer radius and thickness of the annulus is. Or you can describe in the text. Or maybe even put both figures on the same plot so they are scaled the same.

Reply:
I described it in text, didn't realize that GitHub didn't mark is as outdated.